src/gatsby/src/components/advisories/advisoryFilter.js

```diff
@@ -91,21 +105,27 @@ const AdvisoryFilter = ({
             id="event-search-typeahead"
             minLength={0}
             labelKey="label"
+            filterBy={() => true}
+            options={eventTypes}
+            selected={selectedEventType}
+            onChange={(selected) => handleOnChange(selected)}
```

Comment:
I might call this method `handleTypeaheadChange` in this case, since the "on" is kind of redundant